Source/Core/Core/PowerPC/BreakPoints.h

```diff
@@ -69,8 +77,26 @@ class BreakPoints
   void Clear();
   void ClearAllTemporary();
 
+  void SetBreakpointTriggered(bool b, u64 addr = -1, MemCheckCondition cond = MEMCHECK_NONE)
```

Comment:
`b` should have a more descriptive name. Also, you should probably use a std::optional instead of using a special value (-1).